github/github.go

```diff
@@ -681,6 +683,36 @@ func (r *ErrorResponse) Error() string {
 		r.Response.StatusCode, r.Message, r.Errors)
 }
 
+func (r *ErrorResponse) Is(target error) bool {
+	v, ok := target.(*ErrorResponse)
+	if !ok {
+		return false
+	}
+	if (r.Message != v.Message) || (r.DocumentationURL != v.DocumentationURL) {
+		return false
```

Comment:
Please add a unit test that covers this line.

Reply:
Done.